src/create_string.cpp

```diff
@@ -3,12 +3,18 @@
 #include <string.h>
 #include "create_string.h"
 
-char* create_string(Local<Value> value) {
-  if (value->IsNull() || !value->IsString()) {
+char* create_string(Nan::MaybeLocal<v8::Value> maybevalue) {
+  v8::Local<v8::Value> value;
+  
+  if (maybevalue.ToLocal(&value)) {
+    if (value->IsNull() || !value->IsString()) {
+      return 0;
+    }
+  } else {
     return 0;
```

Comment:
Could use the `Maybe` API instead of dangerous null pointers. This would be `return Nan::Nothing();`, the other case `return Nan::Just(str);`